src/UI/Component/Popover/Popover.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace ILIAS\UI\Component\Popover;
+
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Component\Triggerable;
+use ILIAS\UI\Implementation\Component\Popover\ReplaceContentSignal;
+
+/**
+ * Describes the Popover component
+ */
+interface Popover extends Component, Triggerable {
+
+	/**
+	 * Get the title of the popover.
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the components representing the content of the popover.
+	 *
+	 * @return Component[]
+	 */
+	public function getContent();
+
+	/**
+	 * Get the position of the popover.
+	 *
+	 * @return string
```

Comment:
I see your point. Ideally, only the renderer could get the position out of the popover object, not the consumer. I still need a public getter for this case, so it belongs to the interface. Are there any solutions to do this other than introducing the methods `isVerticalPostion()` and `isHorizontalPosition()`?